### PR TITLE
Workload Type3: objects write once and read in iterations using s3api.

### PR DESCRIPTION
**Pull request by DhobaleVishal**
## Problem Statement

-   Problem statement

## Design

-   For Bug, Describe the fix here.
-   For new test addition post the link for design

## Coding

   Checklist for Author

-   [ ]  Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

## Testing

  Checklist for Author

-   [ ]  New/Affected tests are executed
-   [ ]  Attach test execution logs

## Review 

  Checklist for Author

-   [ ]  JIRA number/GitHub Issue added to PR
-   [ ]  PR is self reviewed
-   [ ]  Jira and state/status is updated and JIRA is updated with PR link
-   [ ]  Check if the description is clear and explained

## Documentation

  Checklist for Author

-   [ ]  Changes done to ReadMe

### CODE REVIEW

**Review by ketanarlulkar — APPROVED** [review]

**Review by VikasKumarC — APPROVED** [review]

File: workload/s3/s3api/object_operations_workload_type3.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        - 1Gb
+        - 2Gb
+    write_percentage: 95  # Percentage of data to fill the storage.
+    read_percentage: 100  # Percentage of written data to be used to read from storage.
+    delete_percentage: 0  # Percentage of data to be deleted from storage.
+    cleanup_percentage: 0  # Percentage of data once reached then cleanup all data.
+    #  If None then it will be fetched from cluster(cortx specific.)
+    total_storage_size: None  # Total available storage from cluster or user to be used.
+    min_runtime: 30d  # Minimum execution durations to mark workload pass.
+    sessions_per_node: 20  # Number of sessions per node.
```
Comment (priyankaborawake) [inline workload/s3/s3api/object_operations_workload_type3.yaml]: Lets change this to 2 sessions per node.

File: workload/s3/s3api/object_operations_workload_type3.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    tool: s3api
+    operation: "type3_object_ops"
```
Comment (priyankaborawake) [inline workload/s3/s3api/object_operations_workload_type3.yaml]: We can rename this to something like type3_write_once_read_iterations

**Review by priyankaborawake — APPROVED** [review]
Added minor comments.

File: workload/s3/s3api/object_operations_workload_type3.yaml (hunk 2 above)
Comment (DhobaleVishal) [reply workload/s3/s3api/object_operations_workload_type3.yaml]: done

File: workload/s3/s3api/object_operations_workload_type3.yaml (hunk 1 above)
Comment (DhobaleVishal) [reply workload/s3/s3api/object_operations_workload_type3.yaml]: done

**Pull request merged**